src/styles/utils/bem-block.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  max-width: 200px;
+  width: 100%;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+
+  &__img {
+    width: 100%;
+    margin-bottom: 40px;
+  }
+  &__prod {
+    margin-bottom: 16px;
+  }
+  &__rewie {
```

Comment:
There is a typographical error in the class name 'rewie'. It should be 'review' to maintain consistency with the HTML and avoid potential styling issues.